documentation/docs/getting_started/using_docker.md

```diff
@@ -95,16 +99,16 @@ docker run \
 * `-v $(pwd)/snapshots/mainnet:/app/snapshots/mainnet` Maps the local `snapshots` directory into the container.
 * `--restart always` Instructs Docker to restart the container after Docker reboots.
 * `--name hornet` Name of the running container instance. You can refer to the given container by this name.
-* `--net=host` Instructs Docker to use the host's network, so the network is not isolated. We recommend that you run on host network for better performance.  This way, the container will also open any ports it needs on the host network, so you will not need to specify any ports.
+* `--net=host` Instructs Docker to use the host's network, so the network is not isolated. We recommend that you run on host network for better performance. This way, the container will also open any ports it needs on the host network, so you will not need to specify any ports.
 * `--ulimit nofile=8192:8192` increases the ulimits inside the container. This is important when running with large databases.
 * `-d` Instructs Docker to run the container instance in a detached mode (daemon).
 
 
-You can run `docker stop -t 300 hornet` to gracefully end the process.
+You can run `docker stop -t 300 hornet` to end the process.
```

Comment:
This is actually a technical term, albeit jargo. Ask devs for details.

Reply:
👍